src/containers/shared/css/variables.scss

```diff
@@ -113,33 +113,10 @@ $testnet: $orange-60;
 $devnet: $blue-60;
 $nft-sandbox: $blue-purple-50;
 $custom: $yellow-50;
-$amm: $magenta;
-
-// TX type colors
-// @see global.scss
-$signerlistset: #791aff;
-$setregularkey: #5f00e5;
-$accountdelete: #ff671a;
-$accountset: #ff1a8b;
-$trustset: #afb200;
-$offercreate: #008ae5;
-$offercancel: #ff671a;
-$payment: #00e66a;
-$checkcash: #00803b;
-$checkcancel: #e54d00;
-$checkcreate: #006bb2;
-$escrowcreate: #d91aff;
-$escrowcancel: #e50071;
-$escrowfinish: #c000e5;
-$paymentchannelcreate: #7d8000;
-$paymentchannelfund: #004d23;
-$paymentchannelclaim: #4b4d00;
-$ticketcreate: #55805d;
-$enableamendment: #9ba2b0;
-$setfee: #9ba2b0;
-$unlmodify: #9ba2b0;
+
+// Feature Sets
+$amm: $blue;
```

Comment:
Is this changing for the network as well?

Reply:
It is.